packages/destination-actions/src/destinations/actions-pardot/prospects/index.ts

```diff
@@ -188,28 +188,31 @@ const action: ActionDefinition<Settings, Payload> = {
       return await pa.upsertRecord(payload)
     } catch (err) {
       const error = err as HTTPError
+      if (!error.response) {
+        throw new IntegrationError(`Something went wrong.`, 'PARDOT_ERROR', 500)
```

Comment:
I think we want to log `err` otherwise we still don't know what the real error is. Think `response` object will be null when err is not an HTTPError.

Reply:
Added logger. Thanks!

Reply:
Think it is better if we through the err as such if it is not an instance of HTTPError. Logger is tricky to work with due to the reason mentioned in the previous comment.
```suggestion
        throw err
```